src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { routerUsers } = require('./routes/users');
+const { routerExpenses } = require('./routes/expenses');
+const { userService } = require('./services/users');
+const { expenseService } = require('./services/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+
+  userService.getInitialValue();
+  expenseService.getInitialValue();
+
+  app.use('/users', routerUsers);
+  app.use('/expenses', routerExpenses);
```

Comment:
I'd use here middleware `express.json`, like this:
`app.use('/users', express.json(), userRouter);`